### ISSUE TITLE
Starfinder temporary bonuses do not work.

### ISSUE THREAD

**Issue by CrimsonCastel** [PCGen/pcgen#4162]
In the latest build there is no temporary bonuses for Starfinder

**Comment by LegacyKing** [comment]
I don't recall setting any up... Were there temp bonuses before???

**Comment by Danthekilla** [comment]
Well what I mean is that in pathfinder you can add temporary bonuses and negatives like for instance if you get hasted or level drain.

In starfinder my group cannot find out how to add anything like this as the temporary bonus tab seems to have nothing in it which unfortunately makes it unusable during a session...

Have these just not been setup yet or is starfinder never going to get these?

**Comment by LegacyKing** [comment]
Between work and school, I hardly have any free time to do anything. If you want things, request away. @Nylanfs is wanting me to finish putting together Pathfinder v2 playtest.  I spent maybe 2 hours this weekend plugging away at this and that. >>> Jira System: https://pcgenorg.atlassian.net 

Open whatever requests, and hopefully a volunteer that is less busy than I am will help you out. Otherwise, probably be next weekend when I can cheat some time away.

(Temp bonuses are not really hard to add to be honest. Cheat a few from the existing Pathfinder or 3.5e system. Spell effects are found in the spells TEMPBONUS and VALUE would be the key words you look for [I tended to place the temp bonuses near the end of the files])

Abilities / Templates are the other locations, for things like Rage or Familiars.

Make a list, or feel free to submit the stuff. Either way, my time is up for this weekend.

Cheers,
